src/website/frontend/styles/lib/colors.scss

```
@@ -0,0 +1,110 @@
+$aq-blue-9: #145dff;
+
+$aq-blue-10: #135dff0f;
+
+$aq-blue-11: #edf3ff;
+
+$aq-blue-12: #dfe8f9;
+
+// WHITE
+$aq-white-0: #ffffff;
+
+$aq-white-1: #f5f8ff;
```

Stylelint (17.6.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)